pkg/arm/v6/data/master-startup.sh

```diff
@@ -7,6 +7,62 @@
 # iptables -I INPUT -s $(dig +short "$OS_FQDN" | tail -1) -j LOG --log-prefix "KUBE FQDN DROP:"
 # iptables -I INPUT -s $(dig +short "$OS_FQDN" | tail -1) -j DROP
 
+{{ if .Config.SecurityPatchPackages }}
+logger -t master-startup.sh "installing red hat cdn configuration on $(hostname)"
+cat >/var/lib/yum/client-cert.pem <<'EOF'
+{{ CertAsBytes .Config.Certificates.PackageRepository.Cert | String }}
+EOF
+cat >/var/lib/yum/client-key.pem <<'EOF'
+{{ PrivateKeyAsBytes .Config.Certificates.PackageRepository.Key | String }}
+EOF
+# TODO: delete the following section after all clusters are run with a vm image with kickstart.repo baked in
+cat > /etc/yum.repos.d/kickstart.repo <<'EOF'
+[rhel-7-server-rpms]
+name=Red Hat Enterprise Linux 7 Server (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
+
+[rhel-7-server-extras-rpms]
+name=Red Hat Enterprise Linux 7 Server - Extras (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/extras/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
+
+[rhel-7-server-ose-3.11-rpms]
+name=Red Hat OpenShift Container Platform 3.11 (RPMs)
+baseurl=https://cdn.redhat.com/content/dist/rhel/server/7/7Server/$basearch/ose/3.11/os
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslcacert=/etc/rhsm/ca/redhat-uep.pem
+sslclientcert=/var/lib/yum/client-cert.pem
+sslclientkey=/var/lib/yum/client-key.pem
+enabled=yes
+EOF
+
+logger -t master-startup.sh "installing ARO security updates [{{ StringsJoin .Config.SecurityPatchPackages ", " }}] on $(hostname)"
+while ! yum install -y -q {{ StringsJoin .Config.SecurityPatchPackages " " }}; do
```

Comment:
I don't think the infinite loop here is a good plan.  Suggest trying N (<= 5?) times and failing the script after that

Reply:
i figured as long as red hat CDN isn't down for obscene amounts of time then it wasn't a problem. I'll limit retries to 5.